raiden/tests/integration/fixtures/smartcontracts.py

```diff
@@ -48,7 +55,18 @@ def deploy_all_tokens_register_and_return_their_addresses(
     if register_tokens:
         for token in token_addresses:
             registry = deploy_service.token_network_registry(token_network_registry_address)
-            registry.add_token(token_address=token, given_block_identifier='latest')
+            if contract_manager.contracts_version == DEVELOPMENT_CONTRACT_VERSION:
+                registry.add_token_with_limits(
+                    token_address=token,
+                    channel_participant_deposit_limit=RED_EYES_PER_CHANNEL_PARTICIPANT_LIMIT,
+                    token_network_deposit_limit=RED_EYES_PER_TOKEN_NETWORK_LIMIT,
+                    given_block_identifier='latest',
```

Comment:
As also discussed in the office with @hackaugusto please replace all these `'latest'` with appropriate blockhashes, or make an issue pointing to them and having as the issue's task to do it.